AzureCalling/External/Calling/AudioDeviceType.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import UIKit
+
+enum AudioDeviceType: String {
+    case receiver = "iPhone"
+    case speaker = "Speaker"
+
+    var image: UIImage {
+        switch self {
+        case .receiver:
+            return UIImage(named: "ic_fluent_speaker_2_28_regular")!
+        case .speaker:
+            return UIImage(named: "ic_fluent_speaker_2_28_filled")!
```

Comment:
Just a thought on whether we should set the filled image based on selection instead of device type. For example, receiver is filled and speaker is regular if receiver is selected, whereas speaker is filled and receiver is regular if speaker is selected

Reply:
This can be added in the future if choose to change the icon base on whether it's enabled or not.  We'll need to think a bit more about it however, since I deliberately did not include state in this enum.